src/install/archive/index.md

```diff
@@ -1,18 +1,13 @@
 ---
-layout: default
 title: "Index of Downloads"
 description: "Download specific stable and dev channel versions of the Dart SDK, Dartium, and the Dart API documentation."
 permalink: /install/archive
-
 js:
 - url: /install/archive/out/web/download_archive.dart.js
   defer: true
 - url: /install/archive/assets/install.js
   defer: true
 ---
-
-{% include breadcrumbs.html %}
```

Comment:
Oh, funny that this used to be here (and has done nothing for a long time).

Reply:
Not only that, but the individual pages shouldn't even have this include (the include should only appear in the layout file).